orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/support/TriggerDeserializer.kt

```diff
@@ -31,11 +31,11 @@ import com.netflix.spinnaker.orca.pipeline.model.NexusTrigger
 import com.netflix.spinnaker.orca.pipeline.model.PipelineTrigger
 import com.netflix.spinnaker.orca.pipeline.model.Trigger
 
-internal class TriggerDeserializer :
+class TriggerDeserializer :
```

Comment:
This was needed so we could register custom deserializers into `customTriggerSuppliers` below.